mappings/net/minecraft/server/world/ChunkObservationTracker.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_3210 net/minecraft/server/world/ChunkObservationTracker
```

Comment:
There’s a bit of a mix of watching/observing terminology here. It might be worth using watching since that is more standard through this code.